packages/astro/components/Debug.astro

```diff
@@ -1,6 +1,308 @@
 ---
-import { Prism } from 'astro/components';
+export interface Props {
+  value: any;
```

Comment:
not entirely correct: this describes an interface `<Debug value={...} />`.

Reply:
speaking of: Do we want to support a component with undefined props? Would it be better to do `<Debug value={...} />` instead?

Reply:
We definitely can do `value`. I liked the shorthand option because it exposes your variable name, but I don't feel strongly about either choice.

Reply:
Is there a way to accept either? I like the idea of the shorthand for the same reason!

Reply:
This already does accept either! If you pass `value={anyValue}` it will just print that.

I removed the `export interface Props` because it accepts anything.